documentation/TraceEvent/TraceEventProgrammersGuide.md

```diff
@@ -936,3 +936,41 @@ Unfortunately when a provider registers with the operating system (the Win32 `Ev
 ## On the fly filtering AND Logging using `ETWReloggerTraceEventSource`
 
 TODO
+
+## `UNMANAGED_CODE_TIME` and `CPU_TIME` when analyzing CPU traces collected with EventPipe
```

Comment:
This is useful for people targeting TraceEvent programmatically. For users using PerfView I'd expect them to look here instead: https://github.com/microsoft/perfview/blob/main/src/PerfView/SupportFiles/UsersGuide.htm. That page is the guide that is included in the PerfView tool where all the in-app help links resolve to.

Reply:
Yes, I think that's a better spot for this.